oe_corporate_blocks.post_update.php

```diff
@@ -315,3 +315,15 @@ function oe_corporate_blocks_post_update_40009(): void {
   $config->set('service_navigation', $service_navigation_links);
   $config->save();
 }
+
+/**
+ * Update EU footer data.
+ */
+function oe_corporate_blocks_post_update_40010(&$sandbox): void {
+  // Clear out the institution links from the config.
```

Comment:
I personally would do the following:

```
$config = \Drupal::configFactory()->getEditable('oe_corporate_blocks.eu_data.footer');

if ($config->get('institution_links_title') === 'EU institutions') {
  $config->set('institution_links_title', 'EU institutions and bodies');
}

$config->set('institution_links', [
  [
    'label' => 'Search all EU institutions and bodies',
    'href' => 'https://european-union.europa.eu/institutions-law-budget/institutions-and-bodies/search-all-eu-institutions-and-bodies_en',
  ]
]);
$config->save();
```

This way we minimize the impact on customized footers (there shouldn't be many, but still)

Reply:
We need to remove links from there.